### PR TITLE
create interface MaterialDocument

### PR DESCRIPTION
**Pull request by Hugo-Hontans**
- [x] Create interface MaterialDocument

contributes #1043

### CODE REVIEW

File: apps/catalog/marketplace/marketplace/src/app/movie/selection/selection.component.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export class MarketplaceSelectionComponent implements OnInit {
       territory: detail.territories[0],
       rights: detail.medias[0],
       endRights: (detail.duration as any).to.toDate().toDateString(),
-      languages: detail.languages[0],
+      languages: detail.languages[0] as any
```
Comment (lsenta) [inline apps/catalog/marketplace/marketplace/src/app/movie/selection/selection.component.ts]: `as any`?

File: tsconfig.json [hunk 2]
```diff
@@ -42,7 +42,11 @@
       "@blockframes/invitation/types": ["libs/notification/src/lib/invitation/+state/invitation.firestore.ts"],
 
       "@blockframes/account": ["libs/account/src/index.ts"],
+
+      // Material
       "@blockframes/material": ["libs/material/src/index.ts"],
+      "@blockframes/material/material/types": ["libs/material/src/lib/material/+state/material.firestore.ts"],
```
Comment (lsenta) [inline tsconfig.json]: Inconsistent, check the rest of the file

File: libs/material/src/lib/material/+state/material.model.ts [hunk 3]
```diff
@@ -1,52 +1,22 @@
...
-export interface MaterialRaw {
-  id: string;
-  value: string;
-  description: string;
-  category: string;
-}
+export { MaterialStatus } from './material.firestore';
 
 /** Extends a Material Raw with fields that are specific to Material Template. */
 export interface MaterialTemplate extends MaterialRaw {
```
Comment (lsenta) [inline libs/material/src/lib/material/+state/material.model.ts]: Looks broken, it should extends a `SomethingDocument` from the types.

File: tsconfig.json (hunk 2 above)
Comment (Hugo-Hontans) [reply tsconfig.json]: I will need to create ``@blockframes/materials/delivery/types and @blockframes/material/template/types``

File: libs/material/src/lib/material/+state/material.model.ts (hunk 3 above)
Comment (Hugo-Hontans) [reply libs/material/src/lib/material/+state/material.model.ts]: I have to export MaterialStatus to the lib material without changes.

File: apps/catalog/marketplace/marketplace/src/app/movie/selection/selection.component.ts (hunk 1 above)
Comment (Hugo-Hontans) [reply apps/catalog/marketplace/marketplace/src/app/movie/selection/selection.component.ts]: Hum, I have to change the type of languages in MovieData, I don't really understand  why the type was a string

Comment (Hugo-Hontans) [reply apps/catalog/marketplace/marketplace/src/app/movie/selection/selection.component.ts]: I created an issue to correct that: issue#1066

**Pull request merged**